hazelcast/src/main/java/com/hazelcast/core/HazelcastInstance.java

```diff
@@ -37,9 +37,17 @@
 import java.util.concurrent.ConcurrentMap;
 
 /**
- * Hazelcast instance. Each Hazelcast instance is a member (node) in a cluster.
- * Multiple Hazelcast instances can be created on a JVM.
- * Each Hazelcast instance has its own socket, threads.
+ * Hazelcast instance. Each instance is a member and/or client in a Hazelcast cluster. When
+ * you want to use Hazelcast's distributed data structures, you must first create an instance.
+ * Multiple Hazelcast instances can be created on a single JVM.
+ * 
+ * Instances should be shut down explicitly. When it is a member, use {@link com.hazelcast.core.LifecycleService#shutdown()}.
```

Comment:
HazelcastInstance already has its `shutdown` method and it does the correct thing depending on the instance being client or member. I think we should refer to that method rather than exposing implementation details.